insights/parsers/jboss_version.py

```diff
@@ -49,27 +63,78 @@ def parse_content(self, content):
             "release": int(release)
         }
 
+    @property
+    def product(self):
+        """string: the version of this running JBoss progress."""
+        return self._parsed["product"]
+
     @property
     def version(self):
         """string: the version of this running JBoss progress."""
-        return self.parsed["version"]
+        return self._parsed["version"]
 
     @property
     def major(self):
         """int: the major version of this running JBoss progress."""
-        return self.parsed["major"]
+        return self._parsed["major"]
 
     @property
     def minor(self):
         """int: the minor version of this running JBoss progress."""
-        return self.parsed["minor"]
+        return self._parsed["minor"]
 
     @property
     def release(self):
         """int: release of this running JBoss progress."""
-        return self.parsed["release"]
+        return self._parsed["release"]
 
     @property
     def code_name(self):
         """string: code name of this running JBoss progress."""
-        return self.parsed["code_name"]
+        return self._parsed["code_name"]
+
+
+@parser(Specs.jboss_runtime_versions)
+class JbossRuntimeVersions(Parser):
```

Comment:
Since you are implementing a list here why not just base this class on the `list` type and get all of the functionality for free?  Then you could remove all of the `__fxns__` to make this look like a list.

Reply:
OK. will update to list

Reply:
Updated